package.json

```diff
@@ -17,8 +17,12 @@
   "main": "index.js",
   "scripts": {
     "start": "lerna run --parallel start",
+    "start:demo": "lerna run start --scope @grafana/agent-demo",
     "build": "lerna run build",
     "prepublish": "yarn build",
+    "cypress": "cypress open",
```

Comment:
What I wanted to avoid was having a signle `quality` step in CI - it makes it harder to detect at a glance what failed and get at relevant logs. How about I split into `quality:lint`, `quality:test:unit` and `quality:test:e2e` or smth of that nature, wdyt?